### DISCUSSION TITLE
When creating a new Discussion, show the already created Discussions.

### DISCUSSION THREAD

**Discussion by BoQsc** [community/community#3025] [Discussions]
We might avoid duplication of the discussions way more, this way.

**Comment by evi-liu** [comment]
Thank you for the feedback! A follow-up question: How serious of an issue is duplicate discussions today for your community?

**Comment by PranamBhat** [comment]
In the "Discussions" window just click on "Search all discussions ..." and type any keyword and hit Enter. Already created discussions will list out. Based on that, you can create/decide upon new discussions.

**Comment by hoangmirs** [comment]
In some repositories, there is a required checkbox that warns us to search for similar discussions before starting a new one. But I don't know how to enable it.
https://github.com/orgs/community/discussions/new

![Notification_Center](https://user-images.githubusercontent.com/7344405/189147012-e75fd306-c7b7-43bb-91bb-dd9d74084897.png)
🤔
